Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -84,28 +84,17 @@ void USpatialActorChannel::DeleteEntityIfAuthoritative()
 	UE_LOG(LogTemp, Log, TEXT("Delete entity request on %lld. Has authority: %d"), EntityId, (int)bHasAuthority);
 
 	// If we have authority and aren't trying to delete a critical entity, delete it
-	if (bHasAuthority && !IsCriticalEntity())
+	if (bHasAuthority && !IsSingletonEntity())
 	{
 		Sender->SendDeleteEntityRequest(EntityId);
 		Receiver->CleanupDeletedEntity(EntityId);
 	}
 }
 
-bool USpatialActorChannel::IsCriticalEntity()
+bool USpatialActorChannel::IsSingletonEntity()
```

Comment:
Why was this changed? Do we want to delete the spawner?

Reply:
Or does this never get hit for the spawner?

Reply:
Spawner is no longer an Actor.